jest.config.setup.files.after.env.js

```diff
@@ -1,3 +1,13 @@
+import jestServer from './src/mocks/jest-server.js';
+
+/* eslint-disable no-undef */
```

Comment:
There is a way to fix this: you can make eslint understand that it has Jest loaded when parsing this file. 

But this is fine too.